README.md

```diff
@@ -7,12 +17,29 @@ This is the primary repo for Space Station 14. To prevent people forking RobustT
 If you want to host or create content for SS14, this is the repo you need. It contains both RobustToolbox and the content pack for development of new content packs.
 
 ## Links
+<div class="header" align="center">
 
-[Website](https://spacestation14.io/) | [Discord](https://discord.ss14.io/) | [Forum](https://forum.spacestation14.io/) | [Steam](https://store.steampowered.com/app/1255460/Space_Station_14/) | [Standalone Download](https://spacestation14.io/about/nightlies/)
+[![Website](https://img.shields.io/badge/Website-grey?style=for-the-badge&logo=homepage&logoColor=white)](https://spacestation14.com/)
+[![Discord](https://img.shields.io/discord/310555209753690112?style=for-the-badge&logo=Discord&logoColor=white&label=Discord)](https://discord.ss14.io/)
+[![Forum](https://img.shields.io/badge/Forum-grey?style=for-the-badge&logo=formspree&logoColor=white)](https://forum.spacestation14.com/)
+[![Mastodon Follow](https://img.shields.io/mastodon/follow/109494536058468816?domain=https%3A%2F%2Fmastodon.gamedev.place&style=for-the-badge&logo=Mastodon&logoColor=white)](https://mastodon.gamedev.place/@spacestation14)
+[![Lemmy](https://img.shields.io/badge/Lemmy-grey?style=for-the-badge&logo=lemmy&logoColor=white)](https://lemmy.spacestation14.com/)
+[![patreon](https://img.shields.io/badge/Patreon-grey?style=for-the-badge&logo=patreon&logoColor=white)](https://www.patreon.com/spacestation14)
+[![Steam](https://img.shields.io/badge/Steam-Playtest-g?style=for-the-badge&logo=steam&logoColor=white)](https://store.steampowered.com/app/1255460/Space_Station_14/)
+[![Standalone Download](https://img.shields.io/badge/Standalone_Download-grey?style=for-the-badge&logo=googlecloudstorage&logoColor=white)](https://spacestation14.com/about/nightlies/)
+
+</div>
 
 ## Documentation/Wiki
 
-Our [docs site](https://docs.spacestation14.io/) has documentation on SS14s content, engine, game design and more. We also have lots of resources for new contributors to the project.
+<div class="header" align="center">
+
+[![DevWiki](https://img.shields.io/badge/Space_Wizards_Development_Wiki-grey?style=for-the-badge&logo=mdbook&logoColor=white)](https://docs.spacestation14.com/)
+[![GameWiki](https://img.shields.io/badge/Game_Wiki-grey?style=for-the-badge&logo=gitbook&logoColor=white)](https://wiki.ss14.io/) 
+
+</div>
```

Comment:
Too much information really, looks bad also as shields. Just revert it back to the original with the links updated